UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs

```diff
@@ -179,6 +179,42 @@ public static void LogMessage(string message)
             Debug.Log(message);
         }
 
+        public static IEnumerator SendInfo()
+        {
+            string payload = _jsonInstance.SerializeObject(new GSDKInfo());
+            if (string.IsNullOrEmpty(payload) || string.IsNullOrEmpty(_baseUrl))
+            {
+                yield break;
+            }
+
+            byte[] payloadBytes = Encoding.UTF8.GetBytes(payload);
+
+            if (IsDebugging)
+            {
+                Debug.Log($"state: {CurrentState}, info payload: {payload}");
+            }
+
+            using (UnityWebRequest req = new UnityWebRequest(_baseUrl, UnityWebRequest.kHttpVerbPOST))
+            {
+                req.SetRequestHeader("Accept", "application/json");
+                req.SetRequestHeader("Content-Type", "application/json");
+                req.uploadHandler = new UploadHandlerRaw(payloadBytes) { contentType = "application/json" };
+                yield return req.SendWebRequest();
+
+                if (req.result == UnityWebRequest.Result.ConnectionError || req.result == UnityWebRequest.Result.ProtocolError)
+                {
+                    Guid guid = Guid.NewGuid();
```

Comment:
Why are we creating this guid here? It looks like we'd want some sort of correlationId or something, but this isn't created until after we've already sent the request

Reply:
Hmm, I don't know what that's about. This was copied over from SendHeartbeatRequest and I missed that it isn't useful. Seems it doesn't do anything in the other call either, so I'll take it out of both